client/src/components/backend/Form/HasMany.js

```diff
@@ -10,7 +10,8 @@ export default class FormMakers extends PureComponent {
 
   static propTypes = {
     label: PropTypes.string.isRequired,
-    onNew: PropTypes.func.isRequired,
+    includeCreate: PropTypes.bool,
```

Comment:
Where does this includeCreate prop get used? Is it to disable creation on the maker side of the relationship?